### DISCUSSION TITLE
Historique des messages envoyés aux entreprises + statuts

### DISCUSSION THREAD

**Discussion by Stormfaint** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#34] [Ideas]
Bonjour,

Seriez-vous intéressés par le fait d'ajouter une page d'archive/historique dans l'app society-contacts :
- accessible via un picto à droite du titre comme le "+" pour l'app de SMS
- lister les messages envoyés aux sociétés sur leur répondeur par un joueur sur une page dédiée
- pouvoir avoir un retour sur chacun des messages sur le statut : appel pris / non encore pris + horodatage (envoi + appel pris TBC)
(un peu comme le répondeur entreprise, mais pour les messages envoyés par un joueur donc)

En supplément, pourquoi pas ajouter le fait de préciser via la notification en bas à gauche de l'écran quelle société a pris l'appel.

En tant que joueur (Coby Delany chez UPW), ces features seraient d'une grande aide car nous envoyons de nombreux messages sur les répondeurs des différentes entreprises, et on peut vite s'emmêler les pinceaux, sans compter le suivi quasi impossible.

J'ai un bon background dev, bien que peu familier avec TypeScript, React, Redux et FiveM, mais après avoir un peu audité le code source, ça me semble jouable.

À vous de me dire, et je peux essayer de m'y coller pour proposer quelque chose si ce n'est pas déjà dans votre roadmap.

Aussi, et je doute que ça mérite un PR rien que pour ça : "Status de la cuve:" à remplacer par "Statut de la cuve :" ("status" c'est en anglais, idem pour l'absence d'espace avant les deux points).

Merci pour votre temps !

**Comment by NariieL** [comment]
Hello ! C'est accepté, tu peux tenter de nous proposer quelque chose. L'équipe est favorable à cet ajout.

_N'oublie pas d'y faire une Issue adapté à l'aide d'Improvement, lorsque tu feras ta PR._

Merci beaucoup de t'y être proposé, nous avons hâte de voir cela ! <3

**Comment by jhjuju** [comment]
Bonjour,
Est-ce que tu t'occupes toujours du sujet des messages entreprises @Stormfaint, ou est-ce que je peux le reprendre ?

**Reply by Stormfaint** [reply]
Hello @jhjuju, je n'ai malheureusement absolument pas le temps de m'emparer de ce sujet, donc tu as champ libre si tu le souhaites ! Ça serait génial.
